zigmod.yml

```
@@ -4,4 +4,4 @@ main: src/lib.zig
 license: MIT
 description: Serialization library for JSON
 dependencies:
-  - src: git https://github.com/getty-zig/getty
+  - src: git https://github.com/getty-zig/getty commit-69b4df5
```

```suggestion
  - src: git https://github.com/getty-zig/getty
    version: commit-69b4df5
```

Sorry, I gave you the wrong field to put in.